logger.mjs

```diff
@@ -0,0 +1,25 @@
+import { createLogger, transports, format } from 'winston'
+
+// Imports the Google Cloud client library for Winston
+import { LoggingWinston } from '@google-cloud/logging-winston'
+
+const ENVIRONMENT = process.env["ENVIRONMENT"]
```

Comment:
I'm not sure if you'll have `ENVIRONMENT` available as a env var? If not, you can use the NODE_ENV secret instead, and import it here (https://github.com/trilitech/gcp-terraform-landingzone/blob/main/02-app/terraform/prod/peppermint_startup.sh#L45)

If you do use `NODE_ENV`, you'll have to change `GCP_ENVIRONMENTS` to `['production', 'dev']` (I know this is a bit messy, I'm hoping to make them consistent soon)